test/simple/test-module-version.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+require('../common');
+var assert = require('assert');
```

Comment:
Do you think we might actually want to add `const` in the current trunk? https://github.com/nodejs/node/blob/8ff3d61d8ba90fde01827643db3d87ee97f502e6/test/parallel/test-module-version.js#L3